AGENTS.md

```
@@ -10,42 +9,42 @@ El objetivo es reemplazar el seguimiento en Excel por una tabla pública + panel
 docs/
-  DEV.md # Documentación viva del proyecto (estructura y responsabilidades)
+  DEV.md # Documentación viva del proyecto
+
 tests/ # Unit/Integration tests (Jest)
 tests-e2e/ # E2E tests (Playwright)
-jest.config.mjs # Configuración de Jest (next/jest)
-jest.setup.ts # Setup de Testing Library
-playwright.config.ts # Configuración de Playwright
-
 
 **Reglas de organización**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)